backend/server/src/db/migrations/4-series.sql

```diff
@@ -0,0 +1,20 @@
+call prepare_randomized_ids('series');
+
+create table series (
+    id bigint primary key default randomized_id('series'),
+
+    -- Opencast internal data
+    opencast_id text not null,
+    created_by text not null,
+
+    -- Meta data
+    title text not null,
+    subject text,
+    description text,
+    language text,
+    rights text,
+    license text,
+    organizers text[],
+    contributors text[],
+    publishers text[]
```

Comment:
On `develop.opencast.org` it looks like this shouldn't be an array? I only see `publisher` singular.

Reply:
Are we looking at the same thing? :thinking: 

![image](https://user-images.githubusercontent.com/123272/101800809-f969b480-3b0d-11eb-9dbe-ed9af0006eb8.png)

Reply:
In theory, _all__ metadata can be an array and _could_ hold additional language information. E.g. you could have two titles one in English one in Spanish. In practice, that's never used. I think it's good to keep the fields holding personal identifiers (e.g. names) as an array.

Reply:
Right, I blocked that out. x) But I'm glad to hear that we ignore that, apparently. :sweat_smile: